x-pack/plugins/ml/common/constants/settings.ts

```diff
@@ -5,3 +5,11 @@
  */
 
 export const FILE_DATA_VISUALIZER_MAX_FILE_SIZE = 'ml:fileDataVisualizerMaxFileSize';
+export const ANOMALY_DETECTION_ENABLE_TIME_RANGE = 'ml:anomalyDetection:results:enableTmeDefaults';
```

Comment:
I think there's a letter missing:
```suggestion
export const ANOMALY_DETECTION_ENABLE_TIME_RANGE = 'ml:anomalyDetection:results:enableTimeDefaults';
```

Reply:
Thanks! Fixed in https://github.com/elastic/kibana/pull/76347/commits/ab8b130fd48dd041dac2998f221df63d77d72a4d